pegasus_de.ts

```diff
@@ -34,114 +34,159 @@ in den folgenden Verzeichnissen:</translation>
+        <location filename="../src/frontend/menu/settings/GamepadEditor.qml" line="147"/>
+        <source>Right Y</source>
+        <translation>Rechter Stick Y</translation>
     </message>
     <message>
-        <location filename="../src/frontend/menu/settings/GamepadEditor.qml" line="599"/>
-        <source>guide</source>
-        <translation>Führer</translation>
+        <location filename="../src/frontend/menu/settings/GamepadEditor.qml" line="153"/>
+        <source>Right Press</source>
+        <translation>Rechter Stick drücken</translation>
     </message>
     <message>
-        <location filename="../src/frontend/menu/settings/GamepadEditor.qml" line="618"/>
-        <source>start</source>
-        <translation>Start</translation>
+        <location filename="../src/frontend/menu/settings/GamepadEditor.qml" line="178"/>
+        <source>Settings / Gamepad Layout</source>
+        <translation>Einstellungen / Gamepad-Layout</translation>
     </message>
     <message>
-        <location filename="../src/frontend/menu/settings/GamepadEditor.qml" line="714"/>
-        <source>hold down to quit</source>
-        <translation>zum beenden gedrückt halten</translation>
+        <location filename="../src/frontend/menu/settings/GamepadEditor.qml" line="182"/>
+        <source>No gamepads connected</source>
+        <translation>Keine Gamepads verbunden</translation>
+    </message>
+    <message>
+        <location filename="../src/frontend/menu/settings/GamepadEditor.qml" line="272"/>
+        <source>Hat %1</source>
+        <translation type="unfinished"></translation>
+    </message>
+    <message>
+        <location filename="../src/frontend/menu/settings/GamepadEditor.qml" line="273"/>
+        <source>Button %1</source>
+        <translation>Taste %1</translation>
+    </message>
+    <message>
+        <location filename="../src/frontend/menu/settings/GamepadEditor.qml" line="274"/>
+        <source>Axis %1</source>
+        <translation>Achse %1</translation>
+    </message>
+</context>
+<context>
+    <name>GamepadFooter</name>
+    <message>
+        <location filename="../src/frontend/menu/settings/gamepad/GamepadFooter.qml" line="58"/>
+        <source>Back</source>
+        <translation>Zurück</translation>
+    </message>
+</context>
+<context>
+    <name>GamepadRecorder</name>
+    <message>
+        <location filename="../src/frontend/menu/settings/gamepad/GamepadRecorder.qml" line="107"/>
+        <source>Tilt any axis now to record
+(timeout in %1 seconds)</source>
+        <translation>Zum Festlegen eine beliebige Achse neigen
+        (Zeitüberschreitung in %1 Sekunden)</translation>
+    </message>
+    <message>
+        <location filename="../src/frontend/menu/settings/gamepad/GamepadRecorder.qml" line="108"/>
+        <source>Press any button now to record
+(timeout in %1 seconds)</source>
+        <translation>Zum Festlegen eine beliebige Taste drücken
+        (Zeitüberschreitung in %1 Sekunden)</translation>
```

Comment:
Make sure there's no leading space in the `(Zeitüberschreitung in %1 Sekunden)` lines.

Reply:
sry for the inconvenience. I've updated it and checked if there are other wrong leading spaces